docs/docs/plugins/message-format.mdx

```diff
@@ -18,7 +18,7 @@ npm i messageformat @ngneat/transloco-messageformat
 ## Usage
 The `MessageFormatTranspiler` is compatible with the `DefaultTranspiler` and therefore you can switch without worry that it will break your current translations.
 
-It then enables support for the following within youIt enables support for the following within your i18n translation files:
+It then enables support for the following within your i18n translation files:
```

Comment:
Are you updated with master? this was already fixed

Reply:
This should be resolved now